internal/cmd/marketplace/delete.go

```diff
@@ -34,7 +33,7 @@ const (
 	// deleteItemByTupleEndpointTemplate formatting template for item deletion by the tuple itemID versionID endpoint; specify companyID, itemID, version
 	deleteItemByTupleEndpointTemplate = "/api/backend/marketplace/tenants/%s/resources/%s/versions/%s"
 
-	cmdDeleteAlphaLong = `Delete a single Marketplace item
+	cmdDeleteStableLong = `Delete a single Marketplace item
```

Comment:
non metterei il riferimento a "stable"
```suggestion
	cmdDeleteLongDescription = `Delete a single Marketplace item
```